pkgs/development/python-modules/pwntools/python3.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, Mako, packaging, pysocks, pygments, ROPGadget
+, capstone, paramiko, pip, psutil
+, pyelftools, pyserial, dateutil
+, requests, sortedcontainers, tox, unicorn
+, intervaltree }:
+
+
+buildPythonPackage rec {
+  version = "dev3";
+  pname = "pwntools";
+
+  src = fetchFromGitHub {
+    owner = "Gallopsled";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "130qar90i0vl3s1yqahf3m31mlsmbgj1lnqhdmncwyzhjzbp0x33";
+  };
+
+  propagatedBuildInputs = [
+		Mako packaging pysocks pygments ROPGadget capstone paramiko pip psutil pyelftools pyserial dateutil requests sortedcontainers tox unicorn intervaltree
+	];
+
+  doCheck = false; # no unit tests for the package
```

Comment:
sad :(

Reply:
*c.f.* [powtools/TESTING.md#test-suite](https://github.com/Gallopsled/pwntools/blob/dev3/TESTING.md#test-suite)

They say they have unit tests, but still require a complete environment (with user creation) to run the tests...
Smells like heavy integration tests.

Reply:
it's an integration test, we don't want to do that for every package,